### DISCUSSION TITLE
Detailed information related to long-term aspirin use

### DISCUSSION THREAD

**Discussion by mednlx** [MIT-LCP/mimic-code#1756] [MIMIC-IV]
There are two icd_codes（V5866, Z7982） in the mimic database for long-term aspirin use, and I would like to know the details of this definition, e.g., what is the minimum length of time to be considered "long-term"? What is the minimum dose for long-term use？Your answer would be invaluable to me.

**Comment by alistairewj** [comment]
I would look this up in the ICD coding guidelines since they provide the definitions.

**Reply by mednlx** [reply]
Thank you very much for responding to my question on your busy schedule, I would like you to provide me with the definition of long term aspirin use if you can, thank you very much, thank you!

**Reply by mednlx** [reply]
> 我会在 ICD 编码指南中查找这一点，因为它们提供了定义。

Hello, about taking aspirin for a long time I would like to discuss with you about its definition, please reply me in your busy schedule, it's very important for me, I will express my most sincere thanks to you, good luck!

**Comment by marymlucas** [comment]
Hi @mednlx
I found your question interesting and I hope you don't mind if I try to answer. ICD codes for long term use of a drug do not generally have a specific time or dosage requirement, rather it's based on how and for what they are used. The guidelines on how to select these codes are in the coding guides, for example:

ICD9 - [https://www.ama-assn.org/sites/ama-assn.org/files/corp/media-browser/public/cpt/icd9cm_coding_guidelines_08-09_sm_0.pdf](https://www.ama-assn.org/sites/ama-assn.org/files/corp/media-browser/public/cpt/icd9cm_coding_guidelines_08-09_sm_0.pdf):

"Assign a code from subcategory V58.6, Long-term (current) drug use, if the patient is receiving a medication for an extended period as a prophylactic measure (such as for the prevention of deep vein thrombosis) or as treatment of a chronic condition (such as arthritis) or a disease requiring a lengthy course of treatment (such as cancer). Do not assign a code from subcategory V58.6 for medication being administered for a brief period of time to treat an acute illness or injury (such as a course of antibiotics to treat acute bronchitis)."

And ICD10 - [https://www.cms.gov/files/document/fy-2024-icd-10-cm-coding-guidelines-updated-02/01/2024.pdf](https://www.cms.gov/files/document/fy-2024-icd-10-cm-coding-guidelines-updated-02/01/2024.pdf):

"Assign a code from Z79 if the patient is receiving a medication for an extended period as a prophylactic measure (such as for the prevention of deep vein thrombosis) or as treatment of a chronic condition (such as arthritis) or a disease requiring a lengthy course of treatment (such as cancer). Do not assign a code from category Z79 for medication being administered for a brief period of time to treat an acute illness or injury (such as a course of antibiotics to treat acute bronchitis). "

Hope this helps.

Mary
